external/graphdb-service.js

```diff
@@ -19,7 +19,7 @@ class GraphdbService {
         this.logger = logger;
         this.logger.info(`Data repository name: ${this.config.repositoryName}`);
         const serverConfig = new ServerClientConfig('http://localhost:7200/')
-            .setTimeout(20000)
+            .setTimeout(30000)
```

Comment:
@NZT48 Is this related to numOfRetries?

Reply:
This should be decreased but after we have merged the reinitalization pr into develop. I suggested Samuel to set timeout that he needs to everything start working and later we will decrease it after we have the necessary code updates.

Reply:
Yes, that's the required timeout for the mysql & graphdb initialization during alpine build.